src/Resources/skeleton/authenticator/EmptyAuthenticator.tpl.php

```diff
@@ -3,51 +3,125 @@
 namespace <?= $namespace ?>;
 
 use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\RedirectResponse;
 use Symfony\Component\Security\Core\Authentication\Token\TokenInterface;
 use Symfony\Component\Security\Core\Exception\AuthenticationException;
 use Symfony\Component\Security\Core\User\UserInterface;
 use Symfony\Component\Security\Core\User\UserProviderInterface;
 use Symfony\Component\Security\Guard\AbstractGuardAuthenticator;
 
+/**
+ * Class <?= $class_name ?>
+ * @package <?= $namespace ?>
+ */
 class <?= $class_name ?> extends AbstractGuardAuthenticator
 {
+    /**
+     * Does the authenticator support the given Request?
+     * If this returns false, the authenticator will be skipped.
+     *
+     * @param Request $request
+     * @return bool
+     */
     public function supports(Request $request)
```

Comment:
The @param doesn’t add any value - because that arg is type-hinted. For the @return, can we add a return type instead? I actually can’t temember if that’s allowed - if a class can add a return type even though the method on the interface does not have the return type. But if it’s allowed (I think it is) we should do that instead :)